packages/client/tests/functional/_utils/relationMode/computeMatrix.ts

```diff
@@ -42,9 +46,10 @@ export function computeMatrix({ relationMode, providersDenyList }: ComputeMatrix
   //
   // We skip these combinations in the matrix (= filtering them out)
 
-  const referentialActionsDenylistByProvider = {
+  const referentialActionsDenylistByProviderFlavor = {
     foreignKeys: {
       [Providers.SQLSERVER]: ['Restrict'],
+      [ProviderFlavors.VITESS_8]: referentialActionsBase, // no action is executed for Vitess & relationMode="foreignKeys"
```

Comment:
It's confusing because it can be read as `NO ACTION` which is an action 😄 
```suggestion
      [ProviderFlavors.VITESS_8]: referentialActionsBase, // skip all actions for Vitess & relationMode="foreignKeys"
```

Reply:
But Why do we need to skip all?

Reply:
I improved the comment